docs/oauth/GOOGLE-OAUTH2.md

```diff
@@ -0,0 +1,52 @@
+## Google OAuth2
+We use [vue3-google-oauth2](https://github.com/guruahn/vue3-google-oauth2) Vue package to get the Google sign-in functionality working. This document covers step by step details on how to get the sign-in functionality running.
+
+  - [Set up Google OAuth2 Credentials](#set-up-google-oauth2-credentials)
+  - [Set up Plio Backend API](#set-up-plio-backend-api)
+  - [Testing](#testing)
+  - [In case of failure](#in-case-of-failure)
+  - [Additional Help](#additional-help)
+
+### Set up Google OAuth2 Credentials
+1. Log into the [Google Developer Console](https://console.developers.google.com/)
+2. Click the **Select a Project** dropdown and select **New Project**
+3. Enter the name of the project **Plio**
+4. Once the project is created (takes a few seconds), use the project selector again and select **Plio**
+5. Now you have to add available APIs. Select and open **Library** in the left menu.
+6. Open and Enable **Google+ API** and **Google Analytics API**
+7. Go to the **Credentials** tab, click on the **Create Credentials** popup and select **OAuth client ID** — Google also offers a wizard to help you make this decision if you'd like to use Google Auth in a different context
+8. The API Dashboard opens, click **Credentials** on the left Nav and switch to the **OAuth Consent Screen** tab — provide an application name and an optional logo.
```

Comment:
I had to do step 8 first and then, step 7.

Reply:
Done.